webui/src/data/server.js

```diff
@@ -520,6 +520,11 @@ class ServerCache {
         if (community) {
             q = (q || "") + ' community:' + community;
         }
+        if (drafts) {
+            // TODO: change this once the workflows are working.
+            // Add "submitted" drafts.
+            q = (q ? '(' + q + ') && ' : '') + 'publication_state:draft';
```

Comment:
@emanueldima I changed this. The "&&" operator ensures that "publication_state:draft" filters the query as expected, otherwise it is an "or" and any element in the query will bypass it.

Reply:
You're right, I see now. The same should happen for 'community', the line above